src/App.js

```diff
@@ -60,6 +61,8 @@ class App extends Component {
             <Route path="/settings" exact strict component={() => <Settings googleId={this.state.googleId} />}></Route>
             <Route path={"/" + this.state.username} exact strict component={() => <Profile googleId={this.state.googleId} username={this.state.username} />}></Route>
             <Route path="/:username" exact strict render={({match})=><Profile username={match.params.username}/>}/>
+            <Route path="/:username/movies/:id" exact strict render={({match})=><Movie movieId={match.params.id} username={match.params.username} />}/>
+            <Route path="/settings" exact strict component={() => <Settings googleId={this.state.googleId} />}></Route>
```

Comment:
Why are there two route paths to "/settings"

Reply:
good catch. ill delete one of them